backend/shared/sql/src/SQLWhere.ts

```diff
@@ -11,6 +11,8 @@ export enum SQLWhereSign {
     Less = '<',
     LessEqual = '<=',
     NotEqual = '!=',
+    BooleanMatch = 'BOOLEAN_MATCH',
+    BooleanMatchNot = 'BOOLEAN_MATCH_NOT',
```

Comment:
In plaats van `SQLWhereEqual` te extenden lijkt het mij beter om een totaal nieuwe class te maken die SQLWhere implementeert zoals SQLWhereLike. Dat is netter, want de SQLWhereEqual is echt puur voor vergelijkingen met ==, <, > etc. Terwijl je bij via een custom class (bv. SQLWhereMatch) ook opties kan meegeven over bv. welke 'mode' je wilt gebruiken (boolean of niet). Zo is het ook iets losstaander